src/styles/reset.css

```diff
@@ -0,0 +1,4 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
As far as I know, margines are not set here.

Reply:
yes margin is too heavy for * selector